dot/core/service.go

```diff
@@ -422,12 +423,11 @@ func (s *Service) maintainTransactionPool(block *types.Block) error {
 	// re-validate transactions in the pool and move them to the queue
 	txs := s.transactionState.PendingInPool()
 	for _, tx := range txs {
-		// TODO: re-add this on update to v0.8
-
+		// TODO: re-add this
```

Comment:
does this need to be re-added before merging?

Reply:
I'll open an issue for this, didn't want to do it here